README.md

```diff
@@ -50,7 +50,22 @@ python run.py
 ```
 
 ## Visualisation
-
  For the visualisation of the word embedding model the standalone version of the [tensorflow embedding projector](https://github.com/tensorflow/embedding-projector-standalone) is used.
 Clone the git repository and initiate *newsAnalysis/Projector.py* with the relative path to this repository. 
 *Model.visualise()* automatically loads a trained model to the browser and allows users to explore its words and their relations.
+
```

Comment:
Adapt other README instructions: remove the part about creating a config file and installing dependencies via pip requirements.txt